manifests/a/Avidemux/Avidemux/2.7.8/Avidemux.Avidemux.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Avidemux.Avidemux
+PackageVersion: 2.7.8
+MinimumOSVersion: 10.0.0.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  InstallModes:
+    - interactive
+    - silent
+    - silentWithProgress
+  InstallerSuccessCodes:
+    - 1223
+  InstallerUrl: https://sourceforge.net/projects/avidemux/files/avidemux/2.7.8/avidemux_2.7.8__win64.exe/download
+  InstallerSha256: 5B6197FC98A356E1D3A15163C0813F3AEE7D35FB43D2D8FBE29F5E3A1596A818
```

Let's add the ProductCode as there is no Publisher.

```suggestion
  InstallerSha256: 5B6197FC98A356E1D3A15163C0813F3AEE7D35FB43D2D8FBE29F5E3A1596A818
  ProductCode: "Avidemux 2.7 - 64 bits (64-bit)"
```